src/main/kotlin/org/elm/lang/core/types/TypeRendering.kt

```diff
@@ -56,7 +62,7 @@ private fun StringBuilder.renderLink(refText: String, text: String, linkify: Boo
     else append(text)
 }
 
-
+/** Render a name or destructuring pattern to use a parameter for a function or case branch */
```

Comment:
Yay! I was going to suggest a doc comment here on the last PR but I didn't want to quibble.

But now I _will_ quibble: should be "...to use _as_ a parameter for..."